src/js/services/lang.js

```diff
@@ -65,9 +65,11 @@ const handleChangeLanguageNav = () => {
 /*
   Handle functionality and UI changes for language request input form and new language submission when an input is provided
 */
-const langForm = $('.lang-form');
-
 const languageRequest = () => {
+  const langForm = $('.lang-form');
+  const submitBtn = $('#submit-btn');
+  const input = $('#lang-req')[0];
```

Comment:
`.langForm` and `#lang-req` are effectively the same element so that seems redundant

Reply:
addressed!